code-challenges.js

```diff
@@ -4,60 +4,110 @@
 // Use the test variables provided.
 
 var num1 = 15
-// Expected output: "15 is divisible by three"
+// // Expected output: "15 is divisible by three"
 
 var num2 = 0
 // Expected output: "0 is divisible by three"
 
 var num3 = -7
 // Expected output: "-7 is not divisble by three"
-
-
-
-
+//create a function that will use modulo, first if will check to see if its divisible by 3 and return the number is divisible by three
+const divisibleChecker = (num) => {
+  if (num % 3 === 0){
+    return `${num} is divisible by 3`
+  }
+  //else if will show what will be returned if the number is not divisible by 3
+    else if (num % 3 !== 0) {
+    return `${num} is not divisiable by 3`
+//catch all statement that will show if the code works or not
+  } else {
+    return "broken"
+  }
+}
+//call the code using 3 different console logs
+console.log(divisibleChecker(15));
+console.log(divisibleChecker(0))
+console.log(divisibleChecker(-7))
 
 // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
-
-
-
+//I use the .map method
+//use the .toUpperCase method too
+//console log it
+upperCaseLetters = (randomNouns)
+  for (var i = 0; i < randomNouns.length; i++) {
+    randomNouns[i] = randomNouns.map(i => i.toUpperCase)
+ // return (randomNouns.toUpperCase)
+}
+console.log(randomNouns)
 
 
 // --------------------3) Create a function that takes in the mixedDataArray array and returns an array with ONLY NUMBERS sorted from least to greatest.
 
 var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"]
-// Expected output: [-8, 0, 8, 46, 59, 90, 107]
-
-
-
-
-
-// --------------------4) Create a function that takes in a string and logs the index of the first vowel.
-
+// // Expected output: [-8, 0, 8, 46, 59, 90, 107]
+//
+const noWords = (value) => {
+  let words = [null, undefined, true, false, ""]
+  let array = words.split()
+  return array.filer(value => words.indexOf(value) === -1).join()
+}
+console.log(noWords(mixedDataArray))
```

Comment:
did you run this one on your own computer? i'm not sure what the logic here is trying to achieve. Pseudocoding can really help with that so that i can help you get un-stuck in a review :)